### ISSUE TITLE
change(developer): remove version metadata for keyboards and models from .kps 

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#11892]
https://community.software.sil.org/t/kmp-files-not-updating/8843/8

**Comment by LornaSIL** [comment]
I don't know if this is your intent, but completely removing the version number choice in the .kps seems like a good solution. Then, always use the version from the .kmn.

**Comment by mcdurdin** [comment]
> I don't know if this is your intent, but completely removing the version number choice in the .kps seems like a good solution. Then, always use the version from the .kmn.

Yes, that's my intent -- where FollowKeyboardVersion is set, remove the cached version number before saving.
